modules/ipi-install-troubleshooting-misc-issues.adoc

```diff
@@ -77,6 +77,29 @@ $ kubectl -n openshift-network-operator logs -l "name=network-operator"
 +
 On high availability clusters with three or more control plane (master) nodes, the Operator will perform leader election and all other Operators will sleep. For additional details, see https://github.com/openshift/installer/blob/master/docs/user/troubleshooting.md[Troubleshooting].
 
+== Addressing the `No disk found with matching rootDeviceHints` error
+
+After you deploy a cluster, you might receive the following error:
+
+----
+`No disk found with matching rootDeviceHints`
+----
+
+To address the "No disk found with matching rootDeviceHints" error, a temporary workaround is to change the `rootDeviceHints` to `minSizeGigabytes: 300`.
+
+After you change the `rootDeviceHints`, boot the CoreOS and then verify the disk information by using the following command:
```

Comment:
```suggestion
After you change the `rootDeviceHints` settings, boot the CoreOS and then verify the disk information by using the following command:
```
Per OCP repo guidelines, always include the type of object after the back-ticked object name: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references